src/MiNET/MiNET/Blocks/Grass.cs

```diff
@@ -39,50 +38,38 @@ public partial class Grass : Block
 	{
 		private static readonly ILog Log = LogManager.GetLogger(typeof(Grass));
 
-		public Grass() : base(2)
+		public Grass() : base()
 		{
 			BlastResistance = 3;
 			Hardness = 0.6f;
 		}
 
-		public override void DoPhysics(Level level)
-		{
-			if (level.GameMode == GameMode.Creative) return;
-
-			if (level.GetSubtractedLight(Coordinates.BlockUp()) < 4)
-			{
-				Block dirt = BlockFactory.GetBlockById(3);
-				dirt.Coordinates = Coordinates;
-				level.SetBlock(dirt, true, false, false);
-			}
-		}
```

Comment:
Is there a reason for getting rid of this?

Reply:
there is no logic in Minecraft that would correspond to this action.